src/ElvClient.js

```diff
@@ -147,6 +147,7 @@ class ElvClient {
     fabricURIs,
     ethereumURIs,
     authServiceURIs,
+    searchURIs,
     ethereumContractTimeout = 10,
```

Comment:
Insert corresponding JSDoc comment `* @param {Array<string>} searchURIs - A list of full URIs to search service endpoints` (or `@param {Array<string>=}` if optional

Also looks like there is an old copy-paste error on line 133, should be `* @param {Array<string>} authServiceURIs` ?